packagedb/api.py

```diff
@@ -373,12 +393,19 @@ def index_packages(self, request, *args, **kwargs):
         - unsupported_packages
             - A list of package urls that are not processable by the queue. The
               package indexing queue can only handle npm and maven purls.
+        - unqueued_packages
+            - A list of package urls that were not placed on the queue.
+        - unsupported_vers_count
+            - The number of vers range that are not supported by the univers or package_manager.
+        - unsupported_vers
+            - A list of vers range that are not supported by the univers or package_manager.
         """
-        purls = request.data.getlist('package_urls')
-        unique_purls = set(purls)
+
+        purls = request.data.get('packages')
```

Comment:
Make it more robust with:
```suggestion
        purls = request.data.get('packages') or []
```